src/atoms/button/component.tsx

```diff
@@ -79,19 +80,24 @@ export interface IButtonProps extends IComponentProps<IButtonTheme> {
   onClicked?(): void;
 }
 
-export const Button = (props: IButtonProps): React.ReactElement => {
+export const Button : FC <IButtonProps> = (props: IButtonProps): React.ReactElement => {
```

Comment:
i dont think you need the `: FC<>` here, its implicit. We can add it but we would need to add it everywhere so i think better to exclude for now and we can add later if needed.